src/services/Http.hh

```diff
@@ -0,0 +1,155 @@
+#ifndef RA_SERVICES_HTTP_H
+#define RA_SERVICES_HTTP_H
+#pragma once
+
+#include <functional>
+
+namespace ra {
+namespace services {
+
+class Http
+{
+public:
+    class Response
+    {
+    public:
+        explicit Response(int nStatusCode, const std::string& sResponse) noexcept
+            : m_nStatusCode(nStatusCode), m_sResponse(sResponse)
+        {
+        }
+
+        Response(const Response&) noexcept = default;
+        Response(Response&&) noexcept = default;
```

Comment:
You should probably default the other operations too. Destructor and the assignments. If you didn't want the assignments you should delete them.